packages/nodes-base/nodes/Box/Box.node.ts

```diff
@@ -187,9 +187,9 @@ export class Box implements INodeType {
 							const createdRangeValues = (additionalFields.createdRangeUi as IDataObject)
 								.createdRangeValuesUi as IDataObject;
 							if (createdRangeValues) {
-								qs.created_at_range = `${moment
-									.tz(createdRangeValues.from, timezone)
-									.format()},${moment.tz(createdRangeValues.to, timezone).format()}`;
+								qs.created_at_range = `${moment.tz(createdRangeValues.from, tz).format()},${moment
+									.tz(createdRangeValues.to, tz)
+									.format()}`;
```

Comment:
this formatting still looks not very readable. maybe we should extract these into variables like this:
```suggestion
								const start = moment.tz(createdRangeValues.from, tz).format();
								const ended = moment.tz(createdRangeValues.to, tz).format()
								qs.created_at_range = `${start},${ended}`;
```